library/patient.inc

```diff
@@ -812,6 +814,15 @@ function getPatientPID($args)
 
     return $returnval;
 }
+function getPatientDoBonly($pid)
+{
+    if (empty($pid)) {
+        return "";
+    }
+
+    $sql = sqlQuery("SELECT DOB FROM patient_data WHERE pid = ?", array($pid));
+    return $sql['DOB'];
```

Comment:
Speaking of hair splitting territory... you could add value to this function for future use by return an actual `DateTime` object? It would make the function more versatile.